base_synchro/wizard/base_synchro.py

```diff
@@ -306,14 +314,14 @@ def upload_download(self):
                 self.report.append("No exception.")
             summary = """Here is the synchronization report:
 
-Synchronization started: %s
-Synchronization finished: %s
-
-Synchronized records: %d
-Records updated: %d
-Records created: %d
-
-Exceptions:
+                        Synchronization started: %s
+                        Synchronization finished: %s
+                        
+                        Synchronized records: %d
+                        Records updated: %d
+                        Records created: %d
+                        
+                        Exceptions:
```

Comment:
Have you checked this `summary`?
Please create one record of `res.request` and checked the body of it.